src/env.ts

```diff
@@ -193,12 +193,16 @@ export function setEnvConfiguration(config: Configuration, handlers: FunctionInf
   });
 }
 
-export function setSourceCodeIntegrationEnvVar(handler: ExtendedFunctionDefinition, gitHash: string) {
+export function setSourceCodeIntegrationEnvVar(
+  handler: ExtendedFunctionDefinition,
+  gitHash: string,
+  gitRemote: string,
+) {
   handler.environment ??= {};
   if (handler.environment[ddTagsEnvVar] !== undefined) {
```

Comment:
nit: as we add more and more data, it might be hard to read whether or not it's the exact same string.
maybe check for undefined first to add the comma or not and then append the whole string in every case

Reply:
added your suggestion!